product_vendor_code_search/models/product_product.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    seller_code = fields.Char(
+        compute="_compute_seller_codes", search="_search_seller_code"
```

Comment:
why don't store=true?

Reply:
There's not point in storing the value of this field as it is just a dummy field used to search using the vendor code.

Reply:
But if you do a lot of searches or the field has to be displayed in a view, you'll have to calculate it every time. Keep in mind all this calculation time can slow things down. Do you want to leave it like this then?

Reply:
I don't see why this field would be shown in a view, so I'd leave it like this.